backpack/extensions/firstorder/batch_l2_grad/convtransposend.py

```diff
@@ -1,23 +1,80 @@
+"""batch_l2 extension for ConvTranspose."""
 from torch import einsum
 
-from backpack.extensions.firstorder.base import FirstOrderModuleExtension
+from backpack.extensions.firstorder.batch_l2_grad.batch_l2_base import BatchL2Base
 from backpack.utils import conv_transpose as convTransposeUtils
 
 
-class BatchL2ConvTransposeND(FirstOrderModuleExtension):
+class BatchL2ConvTransposeND(BatchL2Base):
+    """batch_l2 extension for ConvTranspose."""
+
     def __init__(self, N, params=None):
+        """Initialization.
+
+        Args:
+            N: number of dimensions
+            params: list of parameters. Defaults to None.
+        """
         super().__init__(params=params)
         self.N = N
 
     # TODO Use bias Jacobian to compute `bias_gradient`
```

Comment:
Should I keep this TODO?

Reply:
Delete the entire `bias` function as this performs the same steps as the default from the base class.